cms/static/js/views/course_info_update.js

```diff
@@ -218,6 +226,8 @@ define(['codemirror',
                 else {
                     $(this.dateEntry(event)).val('MM/DD/YY');
                 }
+                this.$codeMirrorTitle = CourseInfoHelper.editWithCodeMirror(
+                targetModel, 'content_title', self.options.base_asset_url, $textAreaTitle.get(0));
```

Comment:
is something wrong with indentation here?

Reply:
Fixed, thanks!